internal/entclient/url.go

```diff
@@ -0,0 +1,45 @@
+package entclient
```

Comment:
This file might be better named `devurl.go` to more accurately describe its purpose.

Reply:
Sure, will do